panther_analysis_tool/main.py

```diff
@@ -84,6 +74,14 @@
 SCHEDULED_RULE = "scheduled_rule"
 RULE = "rule"
 
+CUSTOM_FUNCTIONS = [
```

Comment:
Maybe we should call these `RESERVED_FUNCTIONS` or something similar, since they have particular meanings and must return certain types of values if defined.